app/webpack/stylesheets/components/_table.scss

```diff
@@ -197,3 +197,14 @@ td.active {
   background: govuk-colour("light-grey");
   text-align: center;
 }
+
+/* Ensures the table has full width and no margin at the bottom */
+.govuk-table--custom {
+  margin-bottom: 0;
+}
+
+/* Removes the bottom border only from the last row */
+.govuk-table--custom .govuk-table__row--no-bottom-border th,
+.govuk-table--custom .govuk-table__row--no-bottom-border td {
+  border-bottom: none;
```

Comment:
As above, it would be good to use the GDS helpers instead but in this case that might not be possible.